vector/src/main/res/xml/vector_settings_preferences.xml

```diff
@@ -30,6 +30,16 @@
 
     </im.vector.app.core.preference.VectorPreferenceCategory>
 
+    <im.vector.app.core.preference.VectorPreferenceCategory android:title="@string/spaces_header">
```

Comment:
Maybe create a new string for the setting section instead of reusing `spaces_header`?

Reply:
done